src/components/bookings/BookingListMobileView.test.tsx

```diff
@@ -45,7 +51,12 @@ describe('BookingListMobileView', () => {
       isError: true,
     });
 
-    render(<BookingListMobileView />); // Assuming you handle errors inside the component
+    render(
+      <MemoryRouter>
+        {' '}
```

Comment:
Unnecessary space

Reply:
that's happening due to auto saving and formatting. I will be more careful from the next time